pyproject.toml

```diff
@@ -174,6 +174,7 @@ select = [
   "C403",   # https://docs.astral.sh/ruff/rules/unnecessary-list-comprehension-set/
   "C416",   # https://docs.astral.sh/ruff/rules/unnecessary-comprehension/
   "C417",   # https://docs.astral.sh/ruff/rules/unnecessary-map/
+  "C405",   # https://docs.astral.sh/ruff/rules/unnecessary-literal-set/
```

Comment:
Would be nice to keep the rules sorted at least within category - makes it more obvious where gaps are.

Reply:
Agreed.  I was a bit sloppy here because the intent is to fast-follow replace all of this group with a blanket `C` check

https://github.com/wandb/weave/pull/2914/files